docs/Components/BaseComponent.md

```diff
@@ -98,6 +98,18 @@ base_component.add_external_force_function('force1', force1)
 print(base_component.external_forces)
 >>> {'force1': <function force1 at 0x0000025EBCAA8D30>}
 ```
+## *add_external_moment_function*:
+This method is responsible for appending an external moment function to the component and it should return the magnitude of the moment only.
+```python
+def moment1():
+    moment1 = 13
+    return moment1
+
+base_component.add_external_moment_function('moment1', moment1)
+
+print(base_component.external_moments)
+>>> {'moment1': 13}
```

Comment:
Isn't it returning something like ```{'moment1': <function moment1 at 0x7ff87d8891f0>}```?
When we print the function instance, it doesn't evaluate the function (and so returning the value 13).

Reply:
It problably is, but as i'm having the issue with the pytest, i'm not able to run the test and see the exact return of the print without having to manipulate the code to show it. Sorry.

Reply:
No problem. I will try to help you with the pytest problem today.